metricbeat/docs/modules/php_fpm.asciidoc

```diff
@@ -0,0 +1,65 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[metricbeat-module-php_fpm]]
+== PHP-FPM Module
+
+This module periodically fetches metrics from https://php-fpm.org[PHP-FPM]
+servers.
+
+[float]
+=== Module-Specific Configuration Notes
+
+You need to enable the PHP-FPM status page by properly configuring
+`pm.status_path`.
+
+Here is a sample nginx configuration to forward requests to the PHP-FPM status
+page (assuming `pm.status_path` is configured with default value `/status`):
+```nginx
+location ~ /status {
+     allow 127.0.0.1;
+     deny all;
+     include fastcgi_params;
+     fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+     fastcgi_pass 127.0.0.1:9000;
+}
+```
+
+[float]
+=== Compatibility
+
+The PHP-FPM metricsets were tested with PHP 5.6.29 and are expected to
+work with all versions >= 5.
+
+
+[float]
+=== Example Configuration
+
+The php_fpm module supports the standard configuration options that are described
+in <<configuration-metricbeat>>. Here is an example configuration:
+
+[source,yaml]
+----
+metricbeat.modules:
+- module: php_fpm
+  metricsets: ["pool"]
+  enabled: true
+  period: 10s
+  hosts: ["localhost"]
```

Comment:
php-fpm default is `9000` and default status endpoint is `/status`. But these defaults does matter much since it's `fastcgi` protocol and can't be accessed directly. So there is a need to configure a proxy (with `fastcgi` support) and that can be any part and path (no defaults)